### ISSUE TITLE
Crash when create & open assets on macOS (driver issue?)

### ISSUE THREAD

**Issue by mosmetro** [defold/defold#6152]
After the latest security update for macOS 10.15.7 (19H1519) I am unable to create any new or open existing assets: collection, gameobject, tilemap, atlas, etc. -- Defold crashes immediately.

[crash.txt](https://github.com/defold/defold/files/7442088/crash.txt)
[editor2.2021-10-29.log](https://github.com/defold/defold/files/7442089/editor2.2021-10-29.log)

<table>
<tr><td>Build time</td><td>2021-10-18T16:55:48.257691</td></tr>
<tr><td>Defold channel</td><td>editor-alpha</td></tr>
<tr><td>Defold editor sha</td><td>055196fdf29694c06a1000e26217167ac249ba3f</td></tr>
<tr><td>Defold engine sha</td><td>6bfeea3b13d7b8920483ea2cba9c181a8650b84d</td></tr>
<tr><td>Defold version</td><td>1.2.188</td></tr>
<tr><td>GPU</td><td>Intel HD Graphics 4000 OpenGL Engine</td></tr>
<tr><td>GPU Driver</td><td>2.1 INTEL-14.7.20</td></tr>
<tr><td>Java version</td><td>11.0.1+13</td></tr>
<tr><td>OS arch</td><td>x86_64</td></tr>
<tr><td>OS name</td><td>Mac OS X</td></tr>
<tr><td>OS version</td><td>10.15.7</td></tr>
<table>

**Comment by AGulev** [comment]
tit seems related 
 https://forum.defold.com/t/defold-crash-on-air-monterey/69570

**Comment by AGulev** [comment]
https://support.apple.com/en-gb/HT212871

**Comment by AGulev** [comment]
hmm, interesting if this workaround helps https://youtrack.jetbrains.com/issue/JBR-3715

**Comment by dapetcu21** [comment]
@AGulev This does seem to work on Monterey. Thanks! For future reference: In `Defold.app/Contents/Resources/config`, find the line that starts with `vmargs = ` and append `,-XX:+TieredCompilation,-XX:TieredStopAtLevel=1` at the end of it.

**Comment by AGulev** [comment]
👍 i think we should release it as a fix for now, what do you think @britzl ?

**Comment by dapetcu21** [comment]
Nope. False alarm. I still get the same crashes. And I read that thread a bit more and it looks like it has to do with the AArch64 (ARM64) version of Java (that it shouldn't touch a register that the OS is using). This happens to me on x86_64.

**Comment by dapetcu21** [comment]
Also, I feel we're discussing this on the wrong issue. The original post (the 10.15.7 one) has a crash stack that crashes in GL code.

https://forum.defold.com/t/defold-crash-on-air-monterey/69570 and  #6162 look like a Java VM exception in the InvokeLaterDispatcher thread.

**Comment by AGulev** [comment]
oki-doki.

**Comment by britzl** [comment]
I'm able to reproduce this on a MacMini with Catalina and Intel HD Graphics 400:


[crash-macmini-intelhd.txt](https://github.com/defold/defold/files/7538716/crash-macmini-intelhd.txt)
0.

**Comment by britzl** [comment]
https://forum.defold.com/t/defold-crash-on-macos-catalina-10-15-7-19h1519/69712

**Comment by britzl** [comment]
[editor-log-with-jogl-debug.txt](https://github.com/defold/defold/files/7568907/editor-log-with-jogl-debug.txt)
